retroshare-gui/src/gui/PhotoShare/AlbumDialog.cpp

```diff
@@ -39,13 +39,18 @@ AlbumDialog::AlbumDialog(const RsPhotoAlbum& album, TokenQueue* photoQueue, RsPh
 
     mPhotoDrop = ui->scrollAreaWidgetContents;
 
-    if(!(mAlbum.mMeta.mSubscribeFlags & GXS_SERV::GROUP_SUBSCRIBE_ADMIN))
+    if (!(mAlbum.mMeta.mSubscribeFlags & GXS_SERV::GROUP_SUBSCRIBE_ADMIN))
```

Comment:
I just realized that the name is inconsistent: admin rights and subscribe state are totally independent things.